### ISSUE TITLE
Change Player Sprite Mid Game?

### ISSUE THREAD

**Issue by gregorymjason** [rpgboss/rpgboss#478]
In my RPG I would like to switch from character to character as they don't all start out in the same place in the story. So for example, player starts off as character 0, progresses to a certain point in the story, then the player shifts to character 1. I was able to figure out how to edit the party with commands, but how do I use the new character as the player sprite? 

Thanks in advance for the help!!!

**Comment by EgyLynx** [comment]
Only way nowdays is use both C same sprite.

**Comment by gregorymjason** [comment]
? Both C? Is that a code snip it or something? Thanks for the response!

**Comment by TerenasMenethil** [comment]
Yep, you can switch them :) Used the same mechanics in my game at one point. Just make sure that both characters are on your **Database**. 

Create a new event and pick your **Character 1** sprite. Select "**_Modify Party_**". Add Character 1, remove Character 0. By then, your two characters will switch.
